tests/filecheck/projects/riscv-backend-paper/bottom_up.mlir

```diff
@@ -1,23 +1,22 @@
-// RUN: xdsl-opt -p convert-linalg-to-memref-stream,test-optimise-memref-stream,test-lower-memref-stream-to-snitch-stream,test-lower-snitch-stream-to-asm -t riscv-asm %s | filecheck %s
+// RUN: xdsl-opt -p convert-linalg-to-memref-stream,memref-stream-legalize,test-optimise-memref-stream,test-lower-memref-stream-to-snitch-stream,test-lower-snitch-stream-to-asm -t riscv-asm %s | filecheck %s
```

Comment:
Others might disagree, but I view this as part of the scheduling, and not really a separate thing, so I would recommend adding it to `test-optimise-memref-stream` instead of a standalone test in the pipeline

Reply:
Agreed!